src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -685,22 +686,91 @@ internal List<INodeLibraryDependencyInfo> NodeLibraryDependencies
                 foreach (var dependency in value)
                 {
                     //handle package dependencies
-                    if(dependency.ReferenceType == ReferenceType.Package 
+                    if (dependency.ReferenceType == ReferenceType.Package
                         && dependency is PackageDependencyInfo)
                     {
                         foreach (var node in dependency.Nodes)
                         {
                             nodePackageDictionary[node] = (dependency as PackageDependencyInfo).PackageInfo;
                         }
                     }
-                   
                 }
 
                 RaisePropertyChanged(nameof(NodeLibraryDependencies));
             }
         }
 
+        internal List<INodeLibraryDependencyInfo> NodeLocalDefinitions
```

Comment:
Would this be updated when there are new dyf filed imported into current workspace?

Reply:
When the dyf is placed and the user saves the workspace, this will be updated.